run_tests.sh

```diff
@@ -1,11 +1,22 @@
 #!/usr/bin/env bash
 
-images=("php" "fpm" "apache-php" "akeneo-apache")
-versions=("5.6" "5.6" "php-5.6" "php-5.6")
+didFail=
```

Comment:
Agree, but it is as it is done in the official Docker library. I am not a pro of bash scripts, so I kept it like this.